configs/login_server/grd_login.properties

```diff
@@ -22,7 +22,7 @@ loginServer.db.user = root
 loginServer.db.password = 
 loginServer.db.port = 3306
 
-frontend.checkEmailPath = http://localhost/account/checkEmail
+frontend.checkEmailPath = http://localhost/reset
```

Comment:
Better `resetPaaword`?

Reply:
I can't think of any other reset functionality we would want to implement